en/integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi.md

```diff
@@ -0,0 +1,204 @@
+---
+id: applications-monitoring-alyvix-restapi
+title: Alyvix Server
+---
+
+## Overview
+
+Alyvix is an open source APM software tool for visual monitoring.
+
+Build end-user bots visually interacting with any Windows application like ERPs or your favourite browser. 
+Measure end-user experiences: Alyvix records the click-to-appearance responsiveness of each transaction.
+Report IT service quality to support technical and business actions.
+
+The Centreon Plugin-Pack *Alyvix Server* aims to collect the execution status and duration of the Alyvix's *testcases* and their *transactions*
+by requesting the dedicated built-in RestAPI.
+
+> The *Alyvix Server* Plugin-Pack **can not** be used with the Open-Source free version of Alyvix, as the Rest API feature is only
+> available in the commercial version.

```

Comment:
@thibaults-centreon could you please reword to explain that readers have to get in touch with their Centreon sales rep to access to the commercial version please? Idem in FR

Reply:
Please suggest

Reply:
nevermind, I'll push it